fr/administration/secure-platform.md

```diff
@@ -721,3 +721,8 @@ Les journaux des événements Centreon sont disponibles dans les répertoires su
 
 Centreon propose de sauvegarder la configuration de la plateforme. Pour ce faire, accédez au menu
 [`Administration  >  Parameters  >  Backup`](./backup.html).
+
```

Comment:
For me, the shortcut to the getting started chapter should not be present in the security chapter